ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -117,4 +119,8 @@ def _wrapped_api_call(original_func, instance, args, kwargs):
         if request_id2:
             span.set_tag('aws.requestid2', request_id2)
 
+        analytics_sample_rate = config.aiobotocore.get_analytics_sample_rate(use_global_config=False)
+        if analytics_sample_rate:
```

Comment:
We still need to support `analytics_sample_rate = 0` here

Reply:
We might be able to do something like:


```python
span.set_tag(ANALYTICS_SAMPLE_RATE_KEY, config.aiobotocore.get_analytics_sample_rate(use_global_config=False))`
```

And just have our custom code in `set_tag` determine if the value should be set based on the value returned, e.g. `None` means ignore/do not set, `true` means `1.0`, `false` means `0.0`, and any other float value `0 >= x <= 1.0` gets set.